src/schema/nmdc.yaml

```diff
@@ -476,6 +474,24 @@ classes:
         required: false
       part_of:
         required: true
+      extreme_event:
```

Comment:
in mixs.yaml there's also 
"
    from_schema: http://w3id.org/mixs/terms
    source: http://w3id.org/mixs/terms
    see_also:
      - https://github.com/microbiomedata/nmdc-schema/blob/issue-291-mixs-submod/util/rebuild_mixs_yaml.py
   "
@turbomam  should I have left that in there?

Reply:
The new content and your following of SOPs are really good. Here's to many more.